stop_and_wait_plugin/src/stop_and_wait_plugin_node.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2019-2022 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include "stop_and_wait_plugin_node.hpp"
+
+namespace stop_and_wait_plugin
+{
+  namespace std_ph = std::placeholders;
+
+  StopandWaitNode::StopandWaitNode(const rclcpp::NodeOptions &options)
+      : carma_guidance_plugins::TacticalPlugin(options),plugin_name_(get_plugin_name_and_ns()),version_id_("v1.0")
+  {
+    // Create initial config
+    config_ = StopandWaitConfig();
+ 
+    // Declare parameters
+    config_.minimal_trajectory_duration = declare_parameter<std::string>("minimal_trajectory_duration", config_.minimal_trajectory_duration);
+    config_.stop_timestep = declare_parameter<std::string>("stop_timestep", config_.stop_timestep);
+    config_.trajectory_step_size = declare_parameter<std::string>("trajectory_step_size", config_.trajectory_step_size);
+    config_.accel_limit_multiplier = declare_parameter<std::string>("accel_limit_multiplier", config_.accel_limit_multiplier);
+    config_.accel_limit = declare_parameter<std::string>("/vehicle_acceleration_limit", config_.accel_limit);
+    config_.crawl_speed = declare_parameter<std::string>("crawl_speed", config_.crawl_speed);
+    config_.cernterline_sampling_spacing = declare_parameter<std::string>("cernterline_sampling_spacing", config_.cernterline_sampling_spacing);
```

Comment:
let's fix this typo while we are at it
`centerline_sampling_spacing`
Please fix it in .hpp and yaml file as well